packages/demo/src/overlays.ts

```diff
@@ -48,10 +48,10 @@ const overlaysVisibilityButton = document.querySelector('#btn-overlays-visibilit
 const handleOverlaysVisibility = (): void => {
   isOverlaysVisible = !isOverlaysVisible;
   overlaysPlugin.setVisible(isOverlaysVisible);
-  overlaysVisibilityButton.innerText = isOverlaysVisible ? 'Hide overlays' : 'Show overlays';
+  overlaysVisibilityButton.textContent = isOverlaysVisible ? 'Hide overlays' : 'Show overlays';
 };
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-overlaysVisibilityButton.addEventListener('click', _ev => {
+overlaysVisibilityButton.addEventListener('click', _event => {
```

Comment:
**suggestion:** 
```suggestion
overlaysVisibilityButton.addEventListener('click', event => {
```

Reply:
**thought:** This remark applies to all suggestions done on this topic.

Prefixing by `_` is the convention used in TypeScript for unused parameters. We have done this in `bpmn-visualization` to fix SonarCloud code smells in the past.

I use the prefix here because we enforce `noUnusedParameters` (demo) or `strict` (addons, include all noXXX options).
Implementing the suggestion then generates a tsc error: `TS6133:  event  is declared but its value is never read.`